src/assets/scripts/client/aircraft/StripView/StripViewModel.js

```diff
@@ -684,12 +684,28 @@ export default class StripViewModel extends BaseModel {
 
         this.insideCenter = insideCenter;
         this._transponder = transponderCode;
-        this._assignedAltitude = _round(assignedAltitude);
+        this._assignedAltitude = this._roundAltitude(assignedAltitude);
         this._flightPlanAltitude = flightPlanAltitude;
         this._arrivalAirport = arrivalAirportId;
         this._departureAirport = departureAirportId;
         this._flightPlan = flightPlan;
 
         return this._redraw();
     }
+
+    /**
+     * @for StripViewModel
+     * @method roundAltitude
+     * @param altitude
```

Comment:
be sure to add the data type here:
```
@param altitude {number|string}
```

_(assuming there is a string involved due to the `NaN`)_